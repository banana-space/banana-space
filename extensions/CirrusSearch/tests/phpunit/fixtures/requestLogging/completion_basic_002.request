{
	"type": "completion",
	"term": "does not exist or at least i hope not"
}
