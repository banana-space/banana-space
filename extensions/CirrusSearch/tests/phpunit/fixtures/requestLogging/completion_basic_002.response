[{"responses":
    [{
        "took": 28,
        "timed_out": false,
        "_shards": {
            "total": 4,
            "successful": 4,
            "failed": 0
        },
        "hits": {
            "total": 0,
            "max_score": 0,
            "hits": []
        },
        "suggest": {
            "plain": [
                {
                    "text": "does not exist or at least i hope not",
                    "offset": 0,
                    "length": 37,
                    "options": []
                }
            ],
            "plain_fuzzy_1": [
                {
                    "text": "does not exist or at least i hope not",
                    "offset": 0,
                    "length": 37,
                    "options": []
                }
            ],
            "plain_stop": [
                {
                    "text": "does not exist or at least i hope not",
                    "offset": 0,
                    "length": 37,
                    "options": []
                }
            ],
            "plain_stop_fuzzy_1": [
                {
                    "text": "does not exist or at least i hope not",
                    "offset": 0,
                    "length": 37,
                    "options": []
                }
            ]
        }
    }]
}]
