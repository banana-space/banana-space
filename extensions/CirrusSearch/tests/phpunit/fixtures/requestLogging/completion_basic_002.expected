[
    {
        "channel": "cirrussearch-request",
        "context": {
            "$schema": "\/mediawiki\/cirrussearch\/request\/0.0.1",
            "all_elasticsearch_requests_cached": false,
            "elasticsearch_requests": [
                {
                    "cached": false,
                    "hits_offset": 0,
                    "hits_returned": 0,
                    "hits_total": 0,
                    "indices": [
                        "wiki_titlesuggest"
                    ],
                    "max_score": 0,
                    "query": "does not exist or at least i hope not",
                    "query_type": "comp_suggest",
                    "search_time_ms": 0,
                    "suggestion_requested": false
                }
            ],
            "http": {
                "client_ip": "127.0.0.1",
                "has_cookies": false,
                "method": "GET"
            },
            "source": "cli"
        },
        "level": "debug",
        "message": ""
    },
    {
        "channel": "CirrusSearchRequests",
        "context": {
            "elasticTookMs": 0,
            "executor": "123456789",
            "hitsOffset": 0,
            "hitsReturned": 0,
            "hitsTotal": 0,
            "identity": "2c33c9c64851a02e2ba8143c6550a3b8",
            "index": "wiki_titlesuggest",
            "maxScore": 0,
            "query": "does not exist or at least i hope not",
            "queryType": "comp_suggest",
            "source": "cli",
            "tookMs": 0
        },
        "level": "debug",
        "message": "{queryType} search for '{query}' against {index} took {tookMs} millis and {elasticTookMs} Elasticsearch millis. Found {hitsTotal} total results and returned {hitsReturned} of them starting at {hitsOffset}. Requested via {source} for {identity} by executor {executor}"
    }
]